packages/hydrogen-react/codegen.ts

```diff
@@ -1,26 +1,41 @@
 import {CodegenConfig} from '@graphql-codegen/cli';
-import {storefrontApiCustomScalars} from './src/codegen.helpers';
+import {
+  storefrontApiCustomScalars,
+  customerApiCustomScalars,
+} from './src/codegen.helpers';
 
-const config: CodegenConfig = {
-  overwrite: true,
-  schema: {
-    'https://hydrogen-preview.myshopify.com/api/2023-10/graphql.json': {
+const SF_API_VERSION = '2023-10';
+const CA_API_VERSION = '2024-01';
```

Comment:
😬 should we release this in `2024.1.0` to honor the CAAPI version?
Or at least mention explicitly that this is still unstable/release candidate, and that the stable will come in 2024.1.0?

Reply:
I would like to do the update for SFAPI version on a separate PR because there are so much involved (updating hydrogen version and all the doc links) and this PR has enough file changes as it is. I will update the changelog with the unstable information thou!